_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -54,7 +54,7 @@ This rule applies to the first `meta` element in a document for which all the fo
 
 ## Expectation
 
-For each test target, running the [shared declarative refresh steps][], given the target's document, the value of the target's `content` attribute, and the target results in _time_ being either 0 or greater than 72000 (20 hours).
+For each target, the _time_ from the content [attribute value][] is between 0 and 72000 (20 hours). To determine the _time_, run the [shared declarative refresh steps][] on the `meta` element as described in the [HTML refresh state](https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-http-equiv-refresh).
```

Comment:
I do like this formulation. I still think we should specify what are the inputs for the algorithm:
- "a `meta` element _meta_": that is probably unambiguous enough given that the rule targets a `meta` element.
- "[a] string _input_": it is not clear that this string should be the value of the `content` attribute of the target.
- "a `Document` object _document_": is also a bit ambiguous (is it the target's owner document, or its top-level browsing context document?)

Reply:
I disagree. By saying "as described in the HTML refresh state", we're telling you how to get the input needed to run the steps. It's easier to read, and if the input ever changes for whatever reason we have it covered.

Reply:
Ah, yes, I see your point.
It's a bit buried in the third step of the "Will declaratively refresh", but I guess that's worth the simplification 👍